install.sh

```diff
@@ -7,14 +7,19 @@ set -o nounset
 RELEASE_VERSION="latest"
 TEMP_RELEASE_FILE=${TEMP_RELEASE_FILE:=/tmp/aeternity.tgz}
 TARGET_DIR=${TARGET_DIR:=$HOME/aeternity/node}
+SNAPSHOT_DIR=${SNAPSHOT_DIR:=$TARGET_DIR/../maindb}
```

Comment:
the default path should be the node default db_path so that once restored and node started, it starts from the snapshot without additional configuration.

Reply:
c8bad5e